### PR TITLE
feat: Upgraded Login form

### PR DESCRIPTION
**Pull request by SyedImtiyaz-1**
Hello @narayan954 ,
I have upgraded the login form and now it is looking much good.
I have fixed the bugs which was there.

### Fixed List & Upgradation :
- Alignment Issue 
- Overlapping
- Height/Width Issue
- Show Password `svg` Issue
- Added Linear Gradient to the `white background`
- Used Resuable CSS for optimization

### Screenshot 
Light Mode : 
image

DarkMode:
image

### It is ready to Merge 👍🏻

### CODE REVIEW

**Comment by SyedImtiyaz-1** [comment]
@narayan954 kindly review this.

**Comment by narayan954** [comment]
> @narayan954 kindly review this.

will do, seeing some issues, like logo has no left padding/margin.
Also login/signup page is looking bad on mobile view
image

**Comment by SyedImtiyaz-1** [comment]
> > @narayan954 kindly review this.
> 
> will do, seeing some issues, like logo has no left padding/margin. Also login/signup page is looking bad on mobile view image

Okay, will add some media query to fix this issue.

**Comment by SyedImtiyaz-1** [comment]
@narayan954 Working on this give me some time.

**Comment by narayan954** [comment]
> @narayan954 Working on this give me some time.

take your time, I aint forcing anything

**Comment by SyedImtiyaz-1** [comment]
Hello @narayan954 ,
I have fixed the issue.
### Screenshot - 
image

**Review by narayan954 — CHANGES_REQUESTED** [review]
image
how did you check? you current changes are not reflected here

**Comment by SyedImtiyaz-1** [comment]
@narayan954 You can check now.
Previously there is a little issue happened with my internet but now its resolved and I have push the code properly.

**Comment by SyedImtiyaz-1** [comment]
@narayan954 ,
now ok ?

File: src/reusableComponents/Logo/index.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Logo = ({ ml }) => {
   return (
     <p
       id="dummygram-logo"
-      style={{ marginLeft: ml ? ml : "20px" }}
+      style={{ display:"flex", margin:".5vh auto", marginBottom:"2vh"}}
```
Comment (narayan954) [inline src/reusableComponents/Logo/index.jsx:11]: you're modifying a reusable component but still not checking the sideeffect

Comment (narayan954) [reply src/reusableComponents/Logo/index.jsx:11]: nor handling the props

Comment (narayan954) [reply src/reusableComponents/Logo/index.jsx:11]: in fact, the logo is still not in center of login form, and the logo is misbehaving in other parts of the website as well due to this

**Review by narayan954 — CHANGES_REQUESTED** [review]
login form is now misaligned as well. It's missing the bottom margin/padding compulsory for smooth look as earlier. 
- Logo is not aligned properly
- bottom space is lost(most screens)
- main app's css is getting affected/disturbed.
- too much space above(mobile)

Some of visual errors ss:
image
original
image

Changed Bottom
image
original bottom
image

Changed home screen
image
Original home screen
image


closing this pr, and updating the look in some seconds. See if you can update the issue according to the new look

**Comment by narayan954** [comment]
I am merging a new look of these pages, please work on that one if possible

**Pull request closed without merging**